browser/plugins/audio_spatial_listener.plugin.js

```diff
@@ -0,0 +1,95 @@
+(function() {
+
+var SpatialListener = E2.plugins.audio_spatial_listener = function(core, node) 
+{
+	this.desc = 'Spatial Listener.';
+	
+	this.input_slots = [ 
+		{ name: 'position', dt: core.datatypes.VECTOR, desc: 'Position.', def: core.renderer.vector_origin },
+		{ name: 'forward', dt: core.datatypes.VECTOR, desc: 'Forward Vector.', def: core.renderer.vector_origin },
```

Comment:
Default shouldn't be zero. In reality, is this ever anything else than 0,0,-1? In which case do we need this input at all?